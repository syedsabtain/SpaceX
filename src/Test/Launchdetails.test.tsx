
import React from 'react';
import { shallow} from 'enzyme';
import Launchdetails from '../Components/launchdetails/launchdetails';




describe('LaunchDetails',()=>{

    it('div',()=>{
        const data={}
        const com = shallow(<Launchdetails data={data}></Launchdetails>);

        expect(com.find('div').length).toEqual(3)
    });
    it('h4',()=>{

        const data={}
        const com = shallow(<Launchdetails data={data}></Launchdetails>);

        expect(com.find('h4').length).toEqual(4)
    })
});

test('renders the component', () => {

    const data={}
        const com = shallow(<Launchdetails data={data}></Launchdetails>);
    expect(com).toMatchSnapshot();
  
  
  });