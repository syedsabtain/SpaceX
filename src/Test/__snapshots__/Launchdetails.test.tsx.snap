// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<div
  className="mt-3 "
>
  <div
    className=" det"
  >
    <h4>
      Rocket Name : 
      <span
        className="details"
      />
    </h4>
    <h4>
      Mission Name : 
      <span
        className="details"
      />
    </h4>
    <h4>
      Launch Year : 
      <span
        className="details"
      />
    </h4>
    <h4>
      Launch Details : 
      <span
        className="details"
      />
    </h4>
  </div>
  <div
    className="mt-3"
  />
</div>
`;
